luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -196,17 +211,37 @@ class SeqrSVGenotypesSchema(SeqrGenotypesSchema):
 
     def _genotype_fields(self):
         is_called = hl.is_defined(self.mt.GT)
+        was_previously_called = hl.is_defined(self.mt.CONC_ST) & ~self.mt.CONC_ST.contains("EMPTY")
```

Comment:
Occasionally `CONC_ST` is a populated array with the single value of `EMPTY`.  I asked Alba about these.

Reply:
Alba confirmed that `EMPTY` means there's no overlap with the previous callset.